test/integration/fixtures/build-withBabel/src/styled.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+const Title = styled.h1`
+  /* this comment should be removed */
+  font-size: 1.5em;
+  text-align: center;
+  color: palevioletred;
+`;
+
+ReactDOM.render(<Title />, document.getElementById('test'));
```

Comment:
Think I might just be able to `export const Title` and get rid of the `@types/react-dom` dependency (though not `react-dom` itself since it's a dependency of `styled-components)

Reply:
yeppp that worked, also had to `export { Title }` in `index.ts` as it doesn't have side-effects anymore without the render